### ISSUE TITLE
No visible focus when no block selection when entering zoomed out canvas

### ISSUE THREAD

**Issue by jeryj** [WordPress/gutenberg#65830]
### Description

When there is no block selection, the canvas has no visible block selection.

### Step-by-step reproduction instructions

- Reload page so there is no block selection
- Tab to Zoom Out button
- Press Enter to enable zoom out
- Tab to canvas
- No focus is visible

### Screenshots, screen recording, code snippet

https://github.com/user-attachments/assets/616992d0-398b-4f9c-8c24-8947f8ccb136

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes
